communities/aston-css/community-assessment.md

```diff
@@ -0,0 +1,159 @@
+
+### Definition
+Students at Aston University studying/interested in Computer Science or related subjects, and seeking help with the course or to learn more than is covered in their course.
+
+### Size
+
+- How many people are currently in your community?
+
+Around about 120 listed members (although not certain on this), but many do not regularly attend events. Probably about 30 semi-regulars.
+
+- How many would you _like_ to be in your community?
+
+I'd like to get all of those 120 to be attending most of our events!
+
+- Historically, has the group always been about the same size? If not, what brought on that change?
+
+It has been roughly similar as far as I've known, but the active members have declined over the past year due to the previous committee not doing much in the way of interesting/useful events.
+
+### Resources
+
+- What is currently available to people in your community to learn about tech?
+
+There is the course (of course! ᕕ(ȍ‿‿ȍ)ᕗ), but that tends to be pretty dry. There are many members of our society who are good at teaching others, and spreading their knowledge, however that's not frequently utilised. We're aiming to take advantage of this a bit more over the coming year.
+
+- Are there meetups, seminars, online tools, office hours, hackathons?
+
+We frequently attend hackathons as a group (although the turnout is usually not amazing). We went through a phase of hosting office hours (although not by that name), but turnout was often low so we stopped. I think this was mainly because of total lack of structure, and also no real motivation to attend (lots of people do not really do side projects). We also host AstonHack, which gets loads of new people involved!
+
+### Accessibility
+
+- How accessible are the current resources in your community?
+
+Very. Although we are technically a paid society, we allow anybody to attend our events. I'm hoping to keep it this way, despite the student union wanting to enforce everybody being a member.
+
+- Are meetings held during hours where people may be working?
+
+We try to vary when we hold events, but mostly stick with a Wednesday evening (and sometimes Friday evening for socials). Most students shouldn't be working around then!
+
+- Is there childcare available?
+
+No, although this is mostly not applicable to our community. As far as I'm aware this doesn't impact any members.
+
+### Interests
+
+- Is your community made up only of computer science students? Are you OK with that?
+
+We have a majority make up of CS students, but also have members from other engineering subjects and also business students. It might be nice to get a wider range, but business students in particular regularly show little interest in learning much about tech, but are there to find a sidekick for their app idea! This is of course a generalisation, but from experience quite an accurate one at Aston.
```

Comment:
Probably making the purpose of our society clearer and more established so they know that our society isn't a software company (which has to be made clear a lot more than you might think)